test/sampling_correlation_matrices_test.cpp

```diff
@@ -9,18 +9,18 @@
 
 #include "doctest.h"
 #include <iostream>
-#include "misc.h"
+
 #include "random.hpp"
 #include "random/uniform_int.hpp"
 #include "random/normal_distribution.hpp"
 #include "random/uniform_real_distribution.hpp"
 
 #include "cartesian_geom/cartesian_kernel.h"
+#include "diagnostics/univariate_psrf.hpp"
```

Comment:
Do we use psrf in this test (since the samples are matrices)

Reply:
it is used to compute the score few lines below